db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180406080400) do
+ActiveRecord::Schema.define(version: 20180427124945) do
```

Comment:
can you please rebase this [patch](https://github.com/kubic-project/velum/blob/master/packaging/suse/patches/0_set_default_salt_events_alter_time_column_value.rpm.patch) ? otherwise it will fail due to our automated and strict packaging.

this is one thing we always need to do when we update the `schema.rb`